### ISSUE TITLE
Issue with decoding server to server notification

### ISSUE THREAD

**Issue by leearmstrong** [awa/go-iap#107]
I have the following code...

```
var t appstore.SubscriptionNotification
err := json.Unmarshal(body, &t)
```

It gets an error when receiving a S2S from Apple.

> json: cannot unmarshal number into Go struct field NotificationUnifiedReceipt.unified_receipt.status of type string

The status field coming from Apple is `0` - an int.  Not a string!

https://developer.apple.com/documentation/appstoreservernotifications/unified_receipt

Defines it as an Integer

**Comment by jun06t** [comment]
I've released [v1.2.1](https://github.com/awa/go-iap/releases/tag/v1.2.1) :-)
Please use this.

**Comment by leearmstrong** [comment]
Amazing, thanks!
